### PR TITLE
API1600 Support for Volume Attachment

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[API1600 Support for Volume Attachment]

### Issues Resolved
[]

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: spec/unit/resource/api1600/synergy/volume_attachment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1200::Synergy::VolumeAttachment do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1200::C7000::VolumeAttachment' do
+    expect(described_class).to be < OneviewSDK::API1200::C7000::VolumeAttachment
```
Comment (chebroluharika) [inline spec/unit/resource/api1600/synergy/volume_attachment_spec.rb:7]: When I see lib, it seems like this class should inherit from API1600::C7000. If so, please update comment accordingly.

File: spec/unit/resource/api1600/synergy/volume_attachment_spec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1200::Synergy::VolumeAttachment do
```
Comment (chebroluharika) [inline spec/unit/resource/api1600/synergy/volume_attachment_spec.rb:3]: Please correct this.

Comment (nabhajit-ray) [reply spec/unit/resource/api1600/synergy/volume_attachment_spec.rb]: done

File: spec/unit/resource/api1000/synergy/volume_attachment_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1000::Synergy::VolumeAttachment do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1000::C7000::VolumeAttachment' do
+    expect(described_class).to be < OneviewSDK::API1000::C7000::VolumeAttachment
```
Comment (VenkateshRavula) [inline spec/unit/resource/api1000/synergy/volume_attachment_spec.rb:18]: the lib file for API1000::Synergy is inheriting from API800::Synergy. Make check and make changes in spec file accordingly

File: spec/unit/resource/api1200/synergy/volume_attachment_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1200::Synergy::VolumeAttachment do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1200::C7000::VolumeAttachment' do
+    expect(described_class).to be < OneviewSDK::API1200::C7000::VolumeAttachment
```
Comment (VenkateshRavula) [inline spec/unit/resource/api1200/synergy/volume_attachment_spec.rb:18]: Please check the parent class in lib file

File: spec/unit/resource/api800/synergy/volume_attachment_spec.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API800::Synergy::VolumeAttachment do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API800::C7000::VolumeAttachment' do
+    expect(described_class).to be < OneviewSDK::API800::C7000::VolumeAttachment
```
Comment (VenkateshRavula) [inline spec/unit/resource/api800/synergy/volume_attachment_spec.rb:18]: Please check the parent class in lib file

**Pull request merged**